### PR TITLE
Add PHP 8 support

### PR DESCRIPTION
**Pull request by damien-carcel**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2||^8.0",
```
Comment (jmleroux) [inline composer.json:16]:
```suggestion
        "php": ">=7.2",
```
:question:

**Review by jmleroux — APPROVED** [review]
GTM with a suggestion

**Review by pierallard — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (damien-carcel) [reply composer.json:16]: Good question. I know there was a debate about that on Symfony. Dunno the pros and cons.

**Comment by damien-carcel** [comment]
And saddly, I'll have to wait for PHP CS fixer to be compatible with PHP 8…

File: composer.json (hunk 1 above)
Comment (ahocquard) [reply composer.json]: Do you want to support 9.0 in this release? That's why `"^7.2||^8.0"` is better.

**Pull request merged**